external-crates/move/crates/move-compiler/tests/move_check/typing/string_literal_ascii_invalid.snap

```diff
@@ -0,0 +1,30 @@
+---
+source: crates/move-compiler/tests/move_check_testsuite.rs
+info:
+  flavor: core
+  edition: legacy
+  lint: false
+---
+error[E04038]: invalid string after type inference
+  ┌─ tests/move_check/typing/string_literal_ascii_invalid.move:2:39
+  │
+2 │     fun test0(): std::ascii::String { "asdfΓ" }
+  │                                       ^^^^^^^ string "...sdf[\xCE]". contains invalid ASCII entry at char '\xCE' at index '4'
```

Comment:
Should this just read 
```suggestion
  │                                       ^^^^^^^ string "...sdf\xCE". contains invalid ASCII entry at char '\xCE' at index '4'
```

Reply:
My point is that we should (if we don't already) support these hex escape sequences, and so we should just use the syntax that we support in the literal

Reply:
The `[]` were meant to better-indicate the offending character, since span information is a touch hopeless here. That said, removed.